components/style/properties/longhand/ui.mako.rs

```diff
@@ -30,3 +30,61 @@ ${helpers.single_keyword("-moz-window-dragging", "default drag no-drag", product
                          gecko_enum_prefix="StyleWindowDragging",
                          animation_value_type="none",
                          spec="None (Nonstandard Firefox-only property)")}
+
+<%helpers:longhand name="-moz-force-broken-image-icon"
+                   products="gecko"
+                   animation_value_type="none"
+                   spec="None (Nonstandard Firefox-only property)">
+    use cssparser::Token;
+    use std::fmt;
+    use style_traits::ToCss;
+    use values::HasViewportPercentage;
+
+    no_viewport_percentage!(SpecifiedValue);
+
+    pub mod computed_value {
+        #[derive(Debug, Clone, Copy, PartialEq)]
+        #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+        pub struct T(pub bool);
+    }
+
+    pub use self::computed_value::T as SpecifiedValue;
+
+    impl ToCss for computed_value::T {
+        fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+            write!(dest, "{}", self.0 as u8)
```

Comment:
I think this would be a bit clearer like:

```rust
dest.write_str(if self.0 { "1" } else { "0" })
```

but feel free to not address this if you feel differently :)